recipes/jsonnet/all/conanfile.py

```diff
@@ -0,0 +1,71 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+
+
+class JsonnetConan(ConanFile):
+    name = "jsonnet"
+    version = "0.17.0"
```

Comment:
```suggestion
```

Version will be passed by command line, no default version is allowed

Reply:
Removed default version.